### PR TITLE
refactor(frontend): common function to get address from data

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

We use a common function to get the addresses from the `addressStore`, to avoid duplication of code.

### CODE REVIEW

File: src/frontend/src/lib/utils/address.utils.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import type { AddressData } from '$lib/stores/address.store';
+import type { CertifiedStoreData } from '$lib/stores/certified.store';
+import type { Address, OptionAddress } from '$lib/types/address';
+import type { TokenId } from '$lib/types/token';
+
+export const getNullableAddress = ({
```
Comment (peterpeterparker) [inline src/frontend/src/lib/utils/address.utils.ts:6]: The purpose of the function is not really to get a nullable address but rather to map the address regardless if certified or not certified. Do you have a better naming idea? Maybe just `mapAdress` or `addressData`?

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/utils/address.utils.ts:6]: to keep consistent with similar ones (for example the ones related to tokens), I would call it `mapAddress` as you said (or `mapAddressData`)

Comment (peterpeterparker) [reply src/frontend/src/lib/utils/address.utils.ts:6]: Let's do that

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/utils/address.utils.ts]: done

**Review by peterpeterparker — APPROVED** [review]
LGTM, thanks

**Pull request merged**